include/sharg/validators.hpp

```diff
@@ -661,16 +626,9 @@ class output_file_validator : public file_validator_base
 
     /*!\brief The default extensions of `file_t`.
      * \returns A list of default extensions for `file_t`, will be empty if `file_t` is `void`.
-     *
-     * \details
-     *
-     * If `file_t` does name a valid seqan3 file type that contains a static member `valid_formats` returns the
-     * extensions of that `file_t` type. Otherwise returns an empty list.
      */
     static std::vector<std::string> default_extensions()
     {
-        if constexpr (!std::same_as<file_t, void>)
-            return seqan3::detail::valid_file_extensions<typename file_t::valid_formats>();
         return {};
     }
```

Comment:
I think you can remove the entire function as it makes no sense anymore to provide it? This should also be mentioned in the changelog then.

Reply:
But I had to add another function, for not given extensions.